### PR TITLE
Update Check-in Service Providers Integration guide - RAF support

### PR DESCRIPTION
**Pull request by ioigoume**
# Summary

---

**Related issue :**

### CODE REVIEW

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -1025,9 +1025,9 @@ connected to Check-in.
...
-| ----------------------: | :----------------------------------------------------------------- |
-|         **description** | Assurance of the identity of the user                              |
+|  attribute name | Assurance                                        |
+| --------------: | :----------------------------------------------- |
+| **description** | Assurance of the identity of the user, following [https://refeds.org/assurance](REFEDS Assurance Framework (RAF))but MAY use additional assurance frameworks. The following RAF values are qualified and automatically set for all Community identities:<ul><li>$PREFIX$</li><li>$PREFIX$/ID/unique</li><li>$PREFIX$/ID/eppn-unique-no-reassign</li><li>$PREFIX$/IAP/low</li><li>$PREFIX$/ATP/ePA-1m</li><li>$PREFIX$/ATP/ePA-1d</li></ul>Following RAF values are set if the currently used authentication provider asserts (or otherwise qualifies to) them:</br><ul><li>$PREFIX$/IAP/medium</li><li>$PREFIX$/IAP/high</li></ul>The following compound profiles are asserted if the user qualifies to them</br><ul><li>$PREFIX$/profile/cappuccino</li><li>$PREFIX$/profile/espresso</li></ul> |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: Assurance of the identity of the user, following [https://refeds.org/assurance](REFEDS Assurance Framework (RAF)) and the [EGI AAI Assurance Profiles](#level-of-assurance). The following RAF values...

File: content/en/providers/check-in/sp/_index.md [hunk 2]
```diff
@@ -1151,9 +1151,9 @@ urn:mace:egi.eu:aai.egi.eu:vm_operator@fedcloud.egi.eu
 Based on the authentication method selected by the user, the EGI proxy assigns a
 Level of Assurance (LoA), which is conveyed to the SP through both the
 `eduPersonAssurance` attribute and the Authentication Context Class
-(`AuthnContextClassRef`) of the SAML authentication response. EGI AAI currently
-distinguishes between three LoA levels, similarly to the
-[eID Assurance Framework (eIDAF)](http://eur-lex.europa.eu/legal-content/EN/TXT/?uri=OJ:JOL_2015_235_R_0002).
+(`AuthnContextClassRef`) of the SAML authentication response. EGI AAI uses the EGI
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: EGI AAI uses the --> EGI Check-in uses Assurance Profiles which distinguish between three LoA levels, similarly to the...

File: content/en/providers/check-in/sp/_index.md [hunk 3]
```diff
@@ -1163,6 +1163,12 @@ Each level is represented by a URI as follows:
 - **High**: Substantial + multi-factor authn (not yet supported, TBD):
   `https://aai.egi.eu/LoA#High`
 
+Moreover, EGI AAI uses the [REFEDS Assurance framework](https://wiki.refeds.org/display/ASS/REFEDS+Assurance+Framework+ver+1.0).
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: Moreover, EGI AAI --> Moreover, EGI Check-in follows the [REFEDS...

File: content/en/providers/check-in/sp/_index.md [hunk 4]
```diff
@@ -1163,6 +1163,12 @@ Each level is represented by a URI as follows:
...
+The EGI Proxy conveys any RAF values provided by the IdP directly to the SP,
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: The EGI Proxy --> EGI Check-in conveys

File: content/en/providers/check-in/sp/_index.md [hunk 5]
```diff
@@ -1163,6 +1163,12 @@ Each level is represented by a URI as follows:
...
+through the aforementioned methods. In advance, the Proxy will append into the User's
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md]: In advance, the Proxy will append --> Furthermore, Check-in will append

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: content/en/providers/check-in/sp/_index.md (hunk 5 above)
Comment (ioigoume) [reply content/en/providers/check-in/sp/_index.md]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 4 above)
Comment (ioigoume) [reply content/en/providers/check-in/sp/_index.md]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 3 above)
Comment (ioigoume) [reply content/en/providers/check-in/sp/_index.md]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 2 above)
Comment (ioigoume) [reply content/en/providers/check-in/sp/_index.md]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 1 above)
Comment (ioigoume) [reply content/en/providers/check-in/sp/_index.md]: Done

**Comment by ioigoume** [comment]
Closing test PR

**Pull request closed without merging**